infra/gcp/roles/audit.viewer.yaml

```diff
@@ -610,6 +612,12 @@ includedPermissions:
   - consumerprocurement.entitlements.list
   - consumerprocurement.freeTrials.list
   - consumerprocurement.orders.list
+  - contactcenterinsights.analyses.list
+  - contactcenterinsights.conversations.list
+  - contactcenterinsights.issueModels.list
+  - contactcenterinsights.issues.list
+  - contactcenterinsights.operations.list
+  - contactcenterinsights.phraseMatchers.list
```

Comment:
@spiffxp out of curiosity, why do we need this ? Do we have a GCP API use this as a dependency ?

Reply:
I'm happy with the current state of laziness just want to make sure I understand correctly.  
Looking at https://github.com/darkbitio/gcp-iam-role-permissions/blob/master/roles/viewer, I think we should stick at custom roles.